Tests/Parser/Client/fixtures/browser.yml

```diff
@@ -9034,3 +9034,305 @@
+    engine_version: "114.0.5735.196"
+    family: Chrome
+  headers:
+    http-x-requested-with: com.wcd.talkto
+-
+  user_agent: Mozilla/5.0 (Linux; Android 8.1.0; vivo 1817) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/94.0.4606.85 Mobile Safari/537.36
+  client:
+    type: browser
+    name: Proxynet
+    version: ""
+    engine: Blink
+    engine_version: "94.0.4606.85"
+    family: Chrome
+  headers:
+    http-x-requested-with: com.foxylab.airfox
+-
+  user_agent: Mozilla/5.0 (Linux; Android 11; CPH2321 Build/RP1A.200720.011; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/96.0.4664.104 Mobile Safari/537.36
+  client:
+    type: browser
+    name: Good Browser
+    version: ""
+    engine: Blink
+    engine_version: "96.0.4664.104"
+    family: Chrome
+  headers:
+    http-x-requested-with: io.github.mthli.goodbrowser
+-
+  user_agent: Mozilla/5.0 (Linux; Android 13; SM-A536B Build/TP1A.220624.014; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/115.0.5790.138 Mobile Safari/537.36
+  client:
+    type: browser
+    name: Proxyium
+    version: ""
+    engine: Blink
+    engine_version: "115.0.5790.138"
+    family: Chrome
+  headers:
+    http-x-requested-with: app.proxyiumbrowser.android
+-
+  user_agent: Mozilla/5.0 (Linux; Android 10; STK-L21 Build/HUAWEISTK-L21; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/103.0.5060.71 Mobile Safari/537.36
+  client:
+    type: browser
+    name: Vuhuv
+    version: ""
+    engine: Blink
+    engine_version: "103.0.5060.71"
+    family: Chrome
+  headers:
+    http-x-requested-with: com.vuhuv
+-
+  user_agent: Mozilla/5.0 (Linux; Android 7.1.2; LM-X210 Build/N2G47H; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/97.0.4692.98 Mobile Safari/537.36
+  client:
+    type: browser
+    name: BBQ Browser
+    version: ""
+    engine: Blink
+    engine_version: "97.0.4692.98"
+    family: Chrome
+  headers:
+    http-x-requested-with: com.bbqbar.browser
+-
+  user_agent: Mozilla/5.0 (Linux; Android 7.0; Redmi Note 4 Build/NRD90M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/64.0.3282.137 Mobile Safari/537.36
+  client:
+    type: browser
+    name: Fire Browser
+    version: ""
+    engine: Blink
+    engine_version: "64.0.3282.137"
+    family: Chrome
+  headers:
+    http-x-requested-with: com.fast.fireBrowser
+-
+  user_agent: Mozilla/5.0 (Linux; Android 7.0; HTC One M9 Build/NRD90M; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/100.0.4896.127 Mobile Safari/537.36
+  client:
+    type: browser
+    name: Lightning Browser Plus
+    version: ""
+    engine: Blink
+    engine_version: "100.0.4896.127"
+    family: Chrome
+  headers:
+    http-x-requested-with: acr.browser.lightning
+-
+  user_agent: Mozilla/5.0 (Linux; Android 13; Infinix X6716B Build/TP1A.220624.014; wv) AppleWebKit/537.36 (KHTML, like Gecko) Version/4.0 Chrome/108.0.5359.128 Mobile Safari/537.36
+  client:
+    type: browser
+    name: Dark Web
```

Comment:
https://play.google.com/store/apps/details?id=birapp.dark.web.private&hl=en_US does call it "Dark Web Private". Looks to be separate from its https://play.google.com/store/apps/details?id=birapp.dark.web&hl=en_US counterpart.